services/provider/proto/provider.proto

```diff
@@ -169,6 +169,7 @@ message ReportStatusRequest{
 }
 
 message ReportStatusResponse{
-    // Contains subscription channel of provider operator for client operator to match
-    string desiredClientOperatorChannel = 1;
```

Comment:
You cannot remove this, we need to deprecate the field for a couple of releases and only then remove it.
You can add a `DEPRECATED: ` keyword at the start of the comment above the field

Reply:
resolving as i got LGTM in the comment of pr